cloudrone/State state
string user
---
cloudrone/State state
uint32 error